about.html

```diff
@@ -0,0 +1,130 @@
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li><a href="#" class="nav-link">CC Campaign</a></li>
+                
+            </ul>
+
+           
+
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+        </div>
+        </nav>
+    </header>
+
+    <section class="hero-about">
+        <div class="about-hero-wrapper">
+        <p class="hello">"Hello Sharing world</p>
+        <h1>CREATIVE COMMONS <br /> GLOBAL SUMMIT 2015</h1>
+        </div>
+        
+        <p class="about-description">The Global Summit brings together the community of  
+            experts, academics, and activists who comprise the Creative Commons 
+            affiliate network in a 
+            different country every two years.
+            This year, we hope to expand our invitation list including organizations
+             and individuals.</p>
+
+        <div class="support">
+        <p class="message">Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+        <p class="email">ccsummit2015@cckorea.org </p>
+
+        </div>
+    
+    </section>
+    <section class="logo-sub">
+        <h2>Creative Commons <br /> Global Summit 2015 logo</h2>
+        <hr />
+        <p>The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+        <hr />
+        <img class="logo-design" src="./images/about_logo.svg">
+        <hr class="d-line"/>
+
+        <div class="past-summits">
+            <h2>See the past CC Global Summits</h2>
+            <hr />
+            <p>Take a look at the past two
+             CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+             <div class="all-testimonials">
+             <div class="testimonial">
+             <h3 class="year">2011</h3>
+             <p class="about-summit">Global Summit 2011 in Warsaw</p>
+            </div>
+
+             <div class="testimonial">
+             <h3 class="year">2011</h3>
+             <p class="about-summit">Global Summit 2011 in Warsaw</p>
+            </div>
+
+        </div>
+             
+        </div>
+
+    </section>
+
+    <section class="partner">
+        <h2>Partner</h2>
+        <hr />
+        <div class="partner-logos">
+            <img src="./images/airbnb.svg" alt="">
+            <img src="./images/daumkakaologo.svg" alt="">
+            <img src="./images/googlekorea.svg" alt="">
+            <img src="./images/mozilla.svg" alt="">
+            <img src="./images/naver.svg" alt="">
+        </div>
+    </section>
```

Comment:
- [ ] The About page does not have a partners section so I will recommend you remove it so as to stick to the design given to you 👍🏽